docs/external_rules.rst

```diff
@@ -3,20 +3,19 @@
 Including external rules
 ========================
 
-You can include your own custom rules with `-rules` or `--external_rules` arguments. It accepts comma separated list of
-paths to files or directories. Example::
+You can include your own custom rules with ``-rules`` or ``--ext_rules`` arguments.
+It accepts comma separated list of paths to files or directories. Example::
 
     robocop -rules my/own/rule.py -rules rules.py,external_rules.py
 
+Every custom checker (with defined rules) needs to complete following requirements:
```

Comment:
(with defined rules) is kind of  redundant with 2. (also about adding rules class attribute)